pkg/driver/node.go

```diff
@@ -110,7 +111,7 @@ func (d *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	}
 
 	klog.V(5).Infof("NodePublishVolume: creating dir %s", target)
-	if err := d.juicefs.MakeDir(target); err != nil {
+	if err := os.MkdirAll(target, os.FileMode(0755)); err != nil {
```

Comment:
I vaguely remember the abstract driver layer is used to mock API for testing. Does this change affect the unit tests? If not, please ignore.

Reply:
As I upgrade the mount utils package(`k8s.io/utils/mount`), the `fakeJfsProvider` in sanity test also use the `mount.FakeMounter` from `k8s.io/utils/mount`, and the test passed.